client/src/components/AuthHeader/useStyles.ts

```diff
@@ -2,15 +2,15 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   authHeader: {
-    alignSelf: 'flex-end',
-    justifyContent: 'flex-end',
+    alignSelf: 'center',
```

Comment:
I know this is a part of the starter code but most of these styles can be applied to the Box component directly (it is a good practice to pass props to MUI components unless we explicitly need a class to apply some styles) 😄 

Eg: `<Box display="flex" alignItems="center" justifyContent="center"...>`

Reply:
Changes have been made. Can I go ahead and merge this?